### ISSUE TITLE
Consider renaming php-7 to php7-cli (and adding php7-apache)

### ISSUE THREAD

**Issue by aNickzz** [microsoft/vscode-dev-containers#521]
I just spent a while wondering why I couldn't load my app, the problem is that I assumed that php + mariadb meant php-apache + mariadb.

Maybe its just me but it might be beneficial to others too to rename the [php](https://github.com/microsoft/vscode-dev-containers/tree/v0.112.0/containers/php-7) container to php-cli.

Additionally, it would be nice have a version running apache (I guess its as simple as copying the existing php one and changing the `FROM` clause to `php:7-apache` instead of `php:7-cli`, but there's a lot of other stuff in the config / dockerfile that I'm not sure about.

**Comment by Chuxel** [comment]
Thanks for the feedback!  Rather that publishing two sets of images, would it make more sense to just use `-apache`?

**Comment by aNickzz** [comment]
I can't say I've ever seen a cli-only PHP program (other than composer of course) but as a web developer that shouldn't be too surprising.

**Comment by aNickzz** [comment]
I'm probably asking a bit much here, but ideally, there would be versions available for each version of PHP, we have projects still in 7.1 and even 5.3 that are not likely to have the budget for upgrading any time soon.
I'd like to set up a base image I can use for each but I'm not sure what's required in the image to get the full use out of the devcontainer extension. I was going to have a look at the dockerfile for `mcr.microsoft.com/vscode/devcontainers/base:0-focal` but I haven't been able to find it.

**Comment by Chuxel** [comment]
Yeah the I guess the qeustion is the prevalance of just serving using the php command line web server during development e.g. `php -S 0.0.0.0:8000`   Do you just jump into using Apache?

Re: more variations, check out the `base.Dockerfile` - that is what is used to build the image. https://github.com/microsoft/vscode-dev-containers/blob/master/containers/php/.devcontainer/base.Dockerfile  You'll also need the contents of the library-scripts folder since those are used. The build takes longer if you don't have a pre-built image, but the result is the same.

We keep it all together so that you can opt into other scenarios if the base images don't fit your needs.

**Comment by Chuxel** [comment]
I've switched it to use Apache! Should be good to go.
